docs/using-gitbase/functions.md

```diff
@@ -28,6 +37,7 @@ These are all functions that are available because they are implemented in `go-m
 |:-------------|:-------------------------------------------------------------------------------------------------------------------------------|
 |`ARRAY_LENGTH(json)`|if the json representation is an array, this function returns its size.|
 |`AVG(expr)`| returns the average value of expr in all rows.|
+|`BLAME(repository, commit)`|Returns an array, of lines changes and authorship.|
```

Comment:
If this is a gitbase function, it shouldn't be here. It will ve override on the next go-mysql-server update

Reply:
Oh, I misunderstood the previous comment... fixed.